scripts/package

```diff
@@ -5,7 +5,11 @@ cd $(dirname $0)/..
 
 . ./scripts/common
 
-docker run --rm -v `pwd`:/go/src/github.com/rancher/docker-from-scratch dockerscratch-build strip --strip-all build/dockerlaunch
+ID=$(docker run -d dockerscratch-build strip --strip-all build/dockerlaunch)
```

Comment:
Can you move this to build?  I'd like the result of ./script/build be that I get a dockerlaunch binary local on my host